packages/web-react/src/components/Tooltip/useTooltip.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import { ClickEvent } from '../../types';
+
+export interface UseTooltipProps {
+  isDismissible?: boolean;
+}
+
+export interface UseTooltipReturn {
+  open: boolean;
+  onClose: (event: ClickEvent) => void;
+}
+
+export const useTooltip = ({ isDismissible }: UseTooltipProps): UseTooltipReturn => {
+  const [open, setOpen] = useState<boolean>(isDismissible || false);
+
+  const onClose = () => setOpen(false);
```

Comment:
```suggestion
  const handleClose = () => setOpen(false);
```

I suggest using the `handle` prefix for definition of event handlers which are defined inside React components and `on` prefix for the naming of props that transfers those handlers.

`onClose` -> pass function that will be applied on close event
`handleClose` -> event handler that will run on close event

Reply:
ok, fixed